DataFormats/Phase2HLTPFTaus/interface/PFTauPair.h

```diff
@@ -0,0 +1,39 @@
+#ifndef DataFormats_Phase2HLTPFTaus_PFTauPair_h
+#define DataFormats_Phase2HLTPFTaus_PFTauPair_h
+
+#include "DataFormats/Candidate/interface/LeafCandidate.h" // reco::LeafCandidate
+#include "DataFormats/TauReco/interface/PFTau.h"           // reco::PFTau
+#include "DataFormats/TauReco/interface/PFTauFwd.h"        // reco::PFTauRef
+
+namespace reco
+{
+  class PFTauPair : public reco::LeafCandidate
+  {
+   public:
+    PFTauPair();
+    PFTauPair(const reco::PFTauRef& leadPFTau, double leadPFTau_sumChargedIso, const reco::PFTauRef& subleadPFTau, double subleadPFTau_sumChargedIso, double dz);
+    ~PFTauPair();
+
+    const reco::PFTauRef& leadPFTau() const;
+    double leadPFTau_sumChargedIso() const;
```

Comment:
Please use camelCase (preferred), i.e. `leadPFTauSumChargedIso()` or snake_case (e.g. `lead_PFTau_sum_charged_iso()`, but not both at the same time.